src/server/schema.ts

```diff
@@ -2,6 +2,12 @@ export const schema = `
   type Source {
     uri: ID!
     name: String!
+    creators: [Creator]!
```

Comment:
In practice, we only have a single creator for each source in the catalog. Should we be open for extension by having a list here or simplify and keep it a single value?

Reply:
Good question. There are some sources created by more than one organization. But in practice they use the name of their partnership, not their individual names. However, I think we should stick to a flexible data structure in our API and allow for multiple creators.